griffon/commands/queries.py

```diff
@@ -108,6 +108,7 @@ def queries_grp(ctx):
 @progress_bar
 def get_product_summary(ctx, product_stream_name, strict_name_search, all, verbose):
     """get product stream."""
+    ctx.obj["VERBOSE"] = 0
```

Comment:
Why do we set 0 here and on the rest of the places? I thought that verbose (`-v`) option is the top level option and the default value is always 0 according to the `griffon/cli.py::cli`.

Also why it is possible to set `-v` on multiple levels? The is just for convenience so you don't have to always write it on top level?

Reply:
ah, I should review it as a whole, didn't notice it gets deleted afterwards.